### PR TITLE
feat: Improve .NET tag command handling

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

**Comment by jskeet** [comment]
Pinging @busunkim96 for review - I can't assign a reviewer, unfortunately.

**Comment by jskeet** [comment]
cc @amanda-tarafa as this will make more sense of an upcoming PR in google-cloud-dotnet...

**Comment by jskeet** [comment]
I see that the Kokoro failure is due to linting. Will try to clean that up now.

**Comment by jskeet** [comment]
Okay, should be fixed now.

File: releasetool/commands/tag/dotnet.py [hunk 1]
```diff
@@ -58,8 +60,12 @@ def create_releases(ctx: TagContext) -> None:
                 repository=ctx.upstream_repo,
                 tag_name=tag,
                 target_commitish=commitish,
-                name=tag,
-                body=f"Package {package} version {version}",
+                name=f"{package} version {version}",
+                # TODO: either reformat the message as we do in TagReleases,
+                # or make sure we create# the PR with an "already-formatted"
```
Comment (busunkim96) [inline releasetool/commands/tag/dotnet.py:65]: Typo? 
```suggestion
                # or make sure we create the PR with an "already-formatted"
```

Comment (jskeet) [reply releasetool/commands/tag/dotnet.py:65]: Whoops, yes - will fix then merge on green, thanks.

**Comment by jskeet** [comment]
Ah - have now realized I *can't* merge. Please could you do so when you get a chance? Thanks!

**Pull request merged**